src/main/java/cloud/eppo/ConfigurationRequestor.java

```diff
@@ -12,28 +14,44 @@ public class ConfigurationRequestor {
 
   private final EppoHttpClient client;
   private final ConfigurationStore configurationStore;
+  private final Set<String> loadedBanditModelVersions;
 
   public ConfigurationRequestor(ConfigurationStore configurationStore, EppoHttpClient client) {
     this.configurationStore = configurationStore;
     this.client = client;
+    this.loadedBanditModelVersions = new HashSet<>();
   }
 
+  // TODO: async loading for android
   public void load() {
     log.debug("Fetching configuration");
-    Response response = client.get("/api/flag-config/v1/config");
+    String flagConfigurationJsonString = requestBody("/api/flag-config/v1/config");
+    configurationStore.setFlagsFromJsonString(flagConfigurationJsonString);
+
+    Set<String> neededModelVersions = configurationStore.banditModelVersions();
```

Comment:
Is it possible for more than one bandit to have the same model version string?

Reply:
No, model versions are unique to each bandit.

Under the hood, this version ID--for now--is the auto incrementing primary key in a model_versions table that has versions for _all_ bandits (as it includes the bandit id)